lib/registries/addon/components/registration-categories/template.hbs

```diff
@@ -0,0 +1,42 @@
+{{t (concat 'node_categories.' this.selectedCategory)}}
+{{#if @canEdit}}
+    <div local-class='EditMode'>
```

Comment:
That would require `EditableField` to yield the edit button so it can be used as the `<dropdown.trigger>`.  The positioning is also a bit hard to compute given the <dropdown.content>  does not exactly appear underneath the button trigger.

Reply:
Ok, let's not worry about that for now. But it might be worth considering changes to `EditableField` as part of [ENG-394](https://openscience.atlassian.net/browse/ENG-394), to allow more flexibility while providing consistent behavior.